### DISCUSSION TITLE
How about the `ddev debug test` outputs get automatically redacted?

### DISCUSSION THREAD

**Discussion by func0der** [ddev/ddev#6657] [Ideas]
I have just created a `ddev debug test`.
It collected everything from paths, to usernames, computer names, hard disk identifiers etc.

All of those are mostly unique data in and of it self. Combined, the can uniquely identify a persons device.

How about we generalize that data automatically, so that people do not dox (yeah, I am going to get push back on this one ;)) themselves if they want to help? :)

**Comment by rfay** [comment]
I @func0der - We do encourage people to remove data that they consider sensitive.  I think you can understand how much it helps us to understand the host-side and container side information there though. But I certainly consider it a valid idea to redact some things.

We'd love to have you experiment with the script and figure out how to provide the data usefully but hide some of that.

It's a very simple script, at https://github.com/ddev/ddev/blob/master/cmd/ddev/cmd/scripts/test_ddev.sh

There's a contributor training about it at https://ddev.com/blog/ddev-debug-test-contributor-training/

I'd love it if you were to study what it does, figure out the information you consider sensitive and propose a PR to make it less sensitive. 

And by the way, thanks for providing the `ddev debug test`. It does help us so much.
